src/elements/TableCell/TableCell.js

```diff
@@ -182,7 +182,10 @@ const TableCell = ({
     return <td className={cellClassNames}>{data.value}</td>
   } else {
     return (
-      <td className={cellClassNames}>
+      <td
+        data-testid={data?.headerId + (data?.driftStatus ? ' ' + data.driftStatus : '')}
+        className={cellClassNames}
+      >
```

Comment:
That's not correct to use driftStatus here. It should be used outside of this component